martin/src/config/file/main.rs

```diff
@@ -365,6 +393,45 @@ impl Config {
     }
 }
 
+/// Describes the action to take during startup when configuration is found to be invalid
+/// but Martin could still startup in a degraded state (ie, some sources not served).
+#[derive(PartialEq, Eq, Debug, Clone, Copy, Default, Serialize, Deserialize, ValueEnum)]
+#[serde(rename_all = "lowercase")]
+pub enum OnInvalid {
+    /// Log warning messages, abort if the error is critical
+    Warn,
```

Comment:
Two choices (at your liberty).
We shoud have an alias for user-firendlyness though.

```suggestion
    #[serde(alias = "warnings", alias = "warning"))]
    Warn,
```

```suggestion
    #[serde(alias = "warnings", alias = "warning", alias = "continue", alias = "ignore"))]
    Warn,
```